helsingborg-stad/app-mitt-helsingborg
# Feat/summary list

## Feature description
Adds a summary list component.

## Solution description
Adds a new input/view component that summarizes other input fields. Also updates the form-field component to deal with the new components (repeater and summary). 

This is again meant to be merged after previous PRs (eg. stepper poc, repeater-field, Fieldset) because it references and uses various things from these commits. 

## How do I test these changes?
Check the story for Summary List in the storybook. Review the code for the summary list component, and the story to see how it is used. 

## What areas is affected by these changes?.
The new component, and form field. 

## Is there any existing behaviour change of other features due to this code change?
No.

## Covered unit tests cases / E2E test cases?
No.

## Are your code structured in a way so that reviewers can understand it?
Yes, hopefully. The things to send into the summary list is a little involved, but hopefully by looking at prop types and the story people can understand it. 

## Was this feature tested in the following environments?
- [x] On a iOS device/simulator.

Same comment as previously applies: this is supposed to be applied after the RepeaterField PR, and includes those changes. This is not good, and is a result of me not following a good work process with regards to creating branches and separating features enough.
